src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -295,57 +416,42 @@ public void smokeTest() throws Exception {
         c.cleanup(ws);
     }
 
-    private void setGlobalInterpreter(String interpreter) {
-        j.jenkins.getInjector().getInstance(Shell.DescriptorImpl.class).setShell(interpreter);
+    /**
+     * Checks if the golang binary outputs to stdout under normal shell execution.
```

Comment:
Comment copied from #92. Does this test make sense outside of that?

Reply:
Whoops, you are correct. This test has no reason being here. Well, I take that back. One reasoning could be to make sure that the shell wrapper process is not outputting to stdout so that the docker does not wait for it?  Though we know that it is done for certain since ` >&- 2>&-` is applied. Thoughts?

Reply:
> make sure that the shell wrapper process is not outputting to stdout so that the docker does not wait for it

Not sure I understand the concern, but if you think it is important, keep it.